config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
How is code that doesn't even process any input similar to leaving your apartment door open, exactly?